File: 2014-10-CSharp-Fundamentals-1/7. Arrays/PrintArray/PrintArray.cs
using System;

class PrintArrys
{
	static void Main()
	{
		string[] array = {"one", "two", "three", "four"};

		// Process all elements of the array
		for (int index = 0; index < array.Length; index++)
		{
			// Print each element on a separate line
			Console.WriteLine("element[{0}] = {1}", index, array[index]);
		}
	}
}

File: 2015-10-Java-Fundamentals-1/07. Arrays/demos/src/PrintArray.java
import java.text.MessageFormat;

public class PrintArray {
    public static void main(String[] args) {
        String[] array = { "one", "two", "three", "four" };

        // Process all elements of the array
        for (int index = 0; index < array.length; index++) {
            // Print each element on a separate line
            WriteLine("element[{0}] = {1}", index, array[index]);
        }
    }

    public static void WriteLine(String format, Object... obj) {
        MessageFormat form = new MessageFormat(format);
        System.out.println(form.format(obj));
    }
}